projects/Mallard/src/components/issue/issue-row.tsx

```diff
@@ -98,6 +107,19 @@ const IssueButton = ({ issue }: { issue: IssueSummary }) => {
     }, [issue, handleUpdate])
 
     const onDownloadIssue = async () => {
+        switch (downloadBlocked) {
+            case DownloadBlockedStatus.Offline: {
+                Alert.alert('Unable to download', NOT_CONNECTED)
```

Comment:
Shouldn't we give a little more explanation here?

Reply:
Doesn't `NOT_CONNECTED` do this?

Reply:
Ah yes, nevermind!